### PR TITLE
Added smart1 conf and tests

### LINKED ISSUE #16: Add support for Smart1
Look at the tutorials in the docs for guidance on how to begin creating these files: https://astrogeology.usgs.gov/docs/how-to-guides/SPICE/creating-spice-config-files-in-spiceql/

### PR DESCRIPTION
**Pull request by AustinSanders**
Added smart1 conf and tests.

Closes #16

### CODE REVIEW

**Comment by Kelvinrr** [comment]
Add a changelog entry and I think this is good.

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,7 +36,8 @@ release.
 ## [Unreleased]
 
 ### Added
-- Adds support for LO [#11](ihttps://github.com/DOI-USGS/SpiceQL/issues/11)
+- Adds support for LO [#11](https://github.com/DOI-USGS/SpiceQL/issues/11)
+- Adds support for Smart1 [#16](https://github.com/DOI-USGS/SpiceQL/issues/16)
```
Comment (Kelvinrr) [inline CHANGELOG.md:40]: Seems the changelog from another PR was included?

**Review by Kelvinrr — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (AustinSanders) [reply CHANGELOG.md:40]: There was a typo in that line -- it was ihttps:// and this corrects the link

Comment (Kelvinrr) [reply CHANGELOG.md:40]: ah I see now, in the future it should probably be in a separate PR. But we havn't adopted that policy here yet.

**Review by Kelvinrr — APPROVED** [review]

**Pull request merged**